cmd/config/olm/olm.yml

```diff
@@ -65,3 +66,30 @@ jobs:
         inputVars:
           NAME_PREFIX: "benchmark-cc"
           catalogImage: {{.CATALOG_IMAGE}}
+
+  - name: create-clusterextensions
+    namespace: olmv1
+    jobIterations: {{.JOB_ITERATIONS}}
+    maxWaitTimeout: 30m
+    qps: {{.QPS}}
+    burst: {{.BURST}}
+    namespacedIterations: {{.NAMESPACED_ITERATIONS}}
+    iterationsPerNamespace: {{.ITERATIONS_PER_NAMESPACE}}
+    churn: {{.CHURN}}
+    churnCycles: {{.CHURN_CYCLES}}
+    churnDuration: {{.CHURN_DURATION}}
+    churnPercent: {{.CHURN_PERCENT}}
+    churnDelay: {{.CHURN_DELAY}}
+    churnDeletionStrategy: {{.CHURN_DELETION_STRATEGY}}
+    podWait: true
+    waitWhenFinished: true
+    preLoadImages: true
+    preLoadPeriod: 15s
+    objects:
+      - objectTemplate: clusterextension.yml
```

Comment:
The postfix of this filename is not the same as the above file `cmd/config/olm/clusterextension.yaml`.